### DISCUSSION TITLE
Setting up  secure route53 zones

### DISCUSSION THREAD

**Discussion by rverma-dev** [aws-samples/aws-secure-environment-accelerator#701] [Ideas]
Currently ASEA have capabilities for creating route53 hosted zone , but its limited to vpc block only. Similarly we have capabilities of creating kms keys & acm certificates.  Further the target of ASEA is to enforce a good level of security.

Provided this, can we tie up the functionality of creating a secure public-hosted-zone which have dnssec enabled through CMK and a default wildcard acm certificate provisioned.

**Comment by Brian969** [comment]
Worth considering!  Albeit today SEA only orchestrates certificates and ALB's in the SEA home or install region (per the [What we do where chart](https://github.com/aws-samples/aws-secure-environment-accelerator/blob/main/docs/installation/what-we-do-where.md), "Other Security Capabilities" section, lines 6 & 7).  This would require adding support for us-east-1 for certs, and then exploring this functionality.
